src/content/docs/apm/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues.mdx

```diff
@@ -70,17 +87,37 @@ To enable Kafka event collection:
+
+For example:
+
+```
+class_transformer:
+  com.newrelic.instrumentation.kafka-streams-spans-3.2.0:
+    enabled: true
+```
+
+Please refer to the [compatability docs](https://docs.newrelic.com/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent/) to see what instrumentation module you need to enable for your Kafka Streams version.
 
 ## Enable Kafka distributed traces [#collect-kafka-distributed-traces]
 
-The Java agent can also collect [distributed traces](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) from Kafka clients. Enabling traces doesn't affect the agent's normal operations: it will still report metric or event data from Kafka.
+The Java agent can also collect [distributed traces](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) from Kafka clients. Since Kafka Streams runs on top of Kafka Clients, the steps to manage distributed tracing also applies. 
+Enabling traces doesn't affect the agent's normal operations: it will still report metric or event data from Kafka.
 
 Impacts and requirements to consider before enabling:
 
 * **The instrumentation adds a 150 to 200 byte payload to message headers.** If your Kafka messages are very small, traces can add significant processing and storage overhead. This additional payload size could cause Kafka to drop messages if they exceed your Kafka messaging size limit. For this reason, we recommend testing out Kafka distributed traces in a dev environment before enabling them in production.
 * Distributed tracing is only available for Kafka client versions 0.11.0.0 or higher.
 * If you have **not** enabled distributed tracing for your app before, read the [Transition guide](/docs/apm/distributed-tracing/getting-started/transition-guide-distributed-tracing) before enabling.
 * To propagate W3C trace context via Kafka message headers, see the [distributed tracing API usage guide](/docs/agents/java-agent/api-guides/guide-using-java-agent-api#trace-calls) for details on APIs that were released in Java agent 6.4.0. Note that adding additional headers to Kafka messages will further increase the payload size. To see these APIs in action, see [Using Java agent trace APIs with Kafka](https://github.com/newrelic/newrelic-java-examples/tree/main/newrelic-java-agent/distributed-tracing/kafka-examples).
+* If you are using Kafka Streams, you need to enable a spans instrumentation module (refer to the [Kafka Streams transaction section](#collect-kafka-streams-transactions)). Since a transaction is not per record, accepting distributed trace headers is not recomended as it will only work for one record.
 
 The complete process of enabling this is below but at a high level it involves these basic steps: 1) enable tracing via the agent config, and 2) calling the [Java agent API](/docs/agents/java-agent/api-guides/guide-using-java-agent-api) to instrument transactions on both the producer and consumer side.
```

Comment:
```suggestion
The complete process of enabling this is below but at a high level it involves these basic steps: 1) Enable tracing via the agent config, and 2) call the [Java agent API](/docs/agents/java-agent/api-guides/guide-using-java-agent-api) to instrument transactions on both the producer and consumer side.
```

Reply:
This occurs in one sentence in our existing Kafka Docs and isn't a formatted numbered list. I don't think we need to capitalize the letters here.

Reply:
The parenthesis aren't changing it to a formatted numbered list; it's just helps it look cleaner. I also changed the verbs to match in tense (Enable/call). This suggestion is just for consistency/style.

Reply:
If anything, maybe just change it to this: 
```suggestion
The complete process of enabling this is below but at a high level it involves these basic steps: (1) enable tracing via the agent config and (2) call the [Java agent API](/docs/agents/java-agent/api-guides/guide-using-java-agent-api) to instrument transactions on both the producer and consumer side.
```

Reply:
Updated it, thanks!